src/main/java/net/snowflake/client/config/SFClientConfig.java

```diff
@@ -33,6 +43,19 @@ public void setConfigFilePath(String configFilePath) {
     this.configFilePath = configFilePath;
   }
 
+  @SnowflakeJdbcInternalApi
+  public Set<String> getUnknownParamKeys() {
```

Comment:
does it need to be public? we are using it only in the same package

Reply:
Changed